pkg/arvo/app/talk.hoon

```diff
@@ -842,7 +842,23 @@
         ::
           ;~((glue ace) (perk %read ~) cire dem:ag)
         ::
-          ;~((glue ace) (perk %join ~) ;~(plug sorz (punt ;~(pfix ace glyph))))
+          ;~  (glue ace)  (perk %join ~)
+            ;~  pose  sorz
+            ;~  pfix  mic
+            %+  sear
+              |=  num=$@(@ud {p/@u q/@ud})
```

Comment:
I must be missing something here, why is `pick` parsing the number of leading zeroes (for example `;00001`) to p here? I'm just ignoring it.

Reply:
Say you have 250 messages in your talk scrollback. When you do `;5` it doesn't select message nr 5, it selects the last one _ending_ in `5`. In this case, nr 245. If you want to select nr 5, you can do `;005`. It works this way because you're more likely to want recent messages, so those should be more easily accessible.